src/Breadcrumbs/index.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
+import { Breadcrumbs } from "."
+
+
+describe('<Breadcrumbs />', () => {
+    it('should render the steps correctly', () => {
+        const steps = ['breadcrumbs', 'breadcrumbs']
+        render(<Breadcrumbs steps={steps} />)
+
+        for (let s of steps) {
+            const elements = screen.getAllByText(s)
+            for (let e of elements) {
+                expect(e).toBeVisible()
+            }
+        }
```

Comment:
Vc pode validar o comprimento da lista de itens buscados pela query.

```suggestion
const items = await screen.findAllByText(/breadcrumbs/)
expect(items).toHaveLength(2)
```

Acredito que a mesma técnica pode ser aplicada para os outros testes para evitar o uso de loops no teste.